starknet_py/net/signer/stark_curve_signer.py

```diff
@@ -139,3 +139,7 @@ def sign_message(
         # pylint: disable=invalid-name
         r, s = message_signature(msg_hash=msg_hash, priv_key=self.private_key)
         return [r, s]
+
+
+def _is_old_transaction_version(version: int):
+    return version in (0, 2**128)
```

Comment:
This reminds me of the `QUERY_VERSION_BASE` 👀 
https://github.com/software-mansion/starknet.py/pull/649/files#diff-82627af50ed667c0bddd33a217deb26c43ff95cfecca09ec50351d1545e13c81L129

Reply:
I does seem to be the same value, but I'm unsure if should be used here. Do we want this logic to still apply if `QUERY_VERSION_BASE` was updated?